manifest needed to install the data.
the data is also relative- it's in the mrhlpr/data folder, not one up.
--
diff --git a/MANIFEST.in b/MANIFEST.in
new file mode 100644
index 0000000..3a3ce7e
--- /dev/null
+++ b/MANIFEST.in
@@ -0,0 +1 @@
+include mrhlpr/data/*.py
diff --git a/mrhlpr/mr.py b/mrhlpr/mr.py
index 9ca4323..9736cc1 100644
--- a/mrhlpr/mr.py
+++ b/mrhlpr/mr.py
@@ -350,7 +350,7 @@ def fixmsg(mr_id):
     target_branch = get_status(mr_id)["target_branch"]
 
     script = os.path.realpath(os.path.realpath(__file__) +
-                              "/../data/msg_filter.py")
+                              "/data/msg_filter.py")
     os.chdir(git.run(["rev-parse", "--show-toplevel"]))
 
     print("Appending ' (MR " + str(mr_id) + ")' to commits and signing them...")  # noqa: E501
